.trivyignore

```diff
@@ -1,17 +1,13 @@
 # related to telegraf
 #[vishwa] - Fix telegraf & test all for next release - see work item #https://msazure.visualstudio.com/InfrastructureInsights/_workitems/edit/13322134
-CVE-2021-43816
-CVE-2022-23648
-CVE-2022-24450
-CVE-2022-26652
+# Unfixed as of 4/28/2022
 CVE-2019-3826
 CVE-2022-27191
-CVE-2021-42836
 
 # ruby in /usr/lib
-CVE-2020-36327
-CVE-2021-43809
-CVE-2021-41816
-CVE-2021-41819
-CVE-2021-31799
-CVE-2021-28965
\ No newline at end of file
+# CVE-2020-36327
```

Comment:
remove them instead of commenting ? Also, how did you get these resolved? I thought we need to move to 3.* ruby , which i dont see in this PR.

Reply:
oops... should not have been part of this PR. will fix